### PR TITLE
feature/update PowerShell completion

### PR DESCRIPTION
**Pull request by GrooveCS**
# Changes

Please review _DRAFT: Adding current PowerShell Auto Complete functionality offered in spf13/cobra v1.1.13_ #1329 

At the moment, it _Work-in-Progress_, and I would like to complete the submitter checklist over the next several days. 

# Submitter Checklist

These are the criteria that every PR should meet, and please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

_Does your PR contain User facing changes?_
Yes, and you can review in  #1329 issue.

### CODE REVIEW

File: pkg/cmd/completion/completion.go [hunk 1]
```diff
@@ -16,24 +16,24 @@ package completion
...
 `
-	eg = `To load completions:
-
+	examples = `
 Bash:
-
+# Execute the following once:
 $ source <(tkn completion bash)
 
 # To load completions for each session, execute once:
```
Comment (chmouel) [inline pkg/cmd/completion/completion.go:39]: nit: We probably want to mention that this needs to be executed as root to be able to write to /etc/

**Review by chmouel — COMMENTED** [review]
sounds good to me!

thanks

/lgtm

**Comment by chmouel** [comment]
As for the failures, you just need to : 

```
rm -rf vendor/ && make vendor
```

and it should works,

i have compiled it locally and teste it on powershell 7.1.3 for Linux and it works well : 

image

File: pkg/cmd/completion/completion.go (hunk 1 above)
Comment (GrooveCS) [reply pkg/cmd/completion/completion.go:39]: @chmouel thanks for the advice for next time. As a newbie to both golang and this PR workflow. Please provide guidance for next time.

**Comment by GrooveCS** [comment]
> As for the failures, you just need to :
> 
> ```
> rm -rf vendor/ && make vendor
> ```
> 
> and it should works,
> 
> I have compiled it locally and teste it on Powershell 7.1.3 for Linux and it works well :
> 
> image

@chmouel why is `rm -rf vendor/ && make vendor` causing the error in the first place.

Is it due to the go version and complication?

**Comment by chmouel** [comment]
> Is it due to the go version and complication?

@GrooveCS That's how go vendoring works maybe there is other better ways 🤷🏻

**Comment by GrooveCS** [comment]
Merged existing changes and rerun all tests

# Submitter Checklist (**Updated**) 

These are the criteria that every PR should meet, and please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Run the code checkers with `make check.`
- [ ] Regenerate the manpages, docs and go formatting with `make generated.`
- [X] Commit messages to follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

## Notes

1. Reviewed other projects to determine what works
2. Run all of the tests in Ubuntu-20.04 
3. No issues observed

**Comment by GrooveCS** [comment]
Hi @chmouel, 

Do I need to perform any other activities before this needs to be merged? 

Thank you, Parris

**Comment by chmouel** [comment]
Hello, I think there is something wrong with your rebase,

**Comment by GrooveCS** [comment]
Hi @chmouel 

Let me know if I'm closer to doing this correctly?

Thanks, Parris

**Comment by chmouel** [comment]
You are still missing go mod vendor as seens form the job log : 

https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_cli/1339/pull-tekton-cli-build-cross-tests/1381569445465427968/

```$+ ./test/presubmit-tests.sh --build-cross-tests
GOOS=linoux GOARCH=amd64 go build -mod=vendor  -o bin/tkn-linux-amd64 ./cmd/tkn
go: inconsistent vendoring in /home/prow/go/src/github.com/tektoncd/cli:
	github.com/spf13/cobra@v1.1.3: is explicitly required in go.mod, but vendor/modules.txt indicates github.com/spf13/cobra@v1.1.1

run 'go mod vendor' to sync, or use -mod=mod or -mod=readonly to ignore the vendor directory
make: *** [Makefile:44: amd64] Error 1
+ EXIT_VALUE=2
+ set +o xtrace```

**Review by vinamra28 — COMMENTED** [review]
/lgtm

**Review by vdemeester — APPROVED** [review]
/meow

**Pull request merged**